alcs-api/src/application/application.entity.ts

```diff
@@ -54,6 +47,13 @@ export class Application extends Base {
   })
   highPriority: boolean;
 
+  @AutoMap()
+  @Column({
+    type: 'timestamptz',
+    nullable: false,
```

Comment:
if I'm not mistaken nullable is false by default

Reply:
cleaned them up!